gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java

```diff
@@ -68,6 +76,35 @@ public boolean accept(File dir, String name) {
     }
   };
 
+  private static final PathFilter PROPERTIES_PATH_FILTER =        new PathFilter() {
+    @Override
+    public boolean accept(Path path) {
+      String fileExtension = path.getName().substring(path.getName().lastIndexOf('.') + 1);
+      return fileExtension.equalsIgnoreCase(JOB_PROPS_FILE_EXTENSION);
+    }
+  };
+
+  private static final PathFilter NON_PROPERTIES_PATH_FILTER = new PathFilter() {
```

Comment:
This is a negation of `PROPERTIES_PATH_FILTER`, it is generally preferable to explicitly make it a negation. e.g. `accept(Path)` method should be `!PROPERTIES_PATH_FILTER.accept(path)`.